include/onnxruntime/core/session/environment.h

```diff
@@ -81,6 +81,10 @@ class Environment {
 
   Environment() = default;
 
+#ifdef USE_CUDA
+  Status CreateAndRegisterCudaAllocator(const OrtMemoryInfo& mem_info, const std::unordered_map<std::string, std::string>& options, const OrtArenaCfg* arena_cfg = nullptr);
```

Comment:
Please add api description similar to other apis.

Reply:
Also why do we have a CUDA specific one? And will the existing CreateAndRegisterAllocator work for CUDA?